File: src/main/java/yunas/db/DBName.java
package yunas.db;

/**
 * DBName in application.properties
 *
 */
public enum DBName  {

    /**
     * Use yunas.db..... in application.properties
     */
    MASTER("master"),
    /**
     * Use yunas.db.secondary..... in application.properties
     */
    SECONDARY("secondary");

    private String value;


    DBName(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }
}

File: src/main/java/yunas/db/DBName.kt
package yunas.db

/**
 * DBName in application.properties
 *
 */
enum class DBName private constructor(val value: String) {

    /**
     * Use yunas.db..... in application.properties
     */
    MASTER("master"),
    /**
     * Use yunas.db.secondary..... in application.properties
     */
    SECONDARY("secondary")
}
